docs/source/features/mixed_precision.rst

```diff
@@ -3,11 +3,21 @@
 Mixed Precision Training
 ------------------------
 
-Mixed precision training significantly enhances computational efficiency by conducting operations in half-precision and fp8 formats, while selectively maintaining minimal data in single-precision to preserve critical information throughout key areas of the network. NeMo now supports FP16, BF16, and FP8 (via Transformer Engine) across most models. Further details will be provided shortly.
+Mixed precision training significantly enhances computational efficiency by conducting operations in low-precision format, while selectively maintaining minimal data in single-precision to preserve critical information throughout key areas of the network. NeMo now supports FP16, BF16, and FP8 (via Transformer Engine) across most models. Further details will be provided shortly.
 
 
-FP8 usage
-=========
+Half-precision Training
+=======================
+
+NeMo supports half-precision (FP16 and BF16) computation training via Megatron Core and the distributed optimizer.
+This training recipe uses half-precision in all layer computation keeping the model states (optimizer states, master parameters, and parameter gradients) in single-precision.
```

Comment:
is it always true we keep a single precision copy, no args to turn off?

Reply:
A few lines later you mention that parameter gradients are stored in half-precision, which is inconsistent with the statement above (single precision). In generals, param grads are in half precision when O2 is on, so you assume O2 is switched off in the statement above? It'd be great if you could clarify.